packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -157,6 +169,23 @@ export default declare((api, options) => {
     },
   };
 
+  const staticPrivatePropertyHandler = {
+    handle(member) {
```

Comment:
So this doesn't handle `get`, `set`, and `call` operations. Given that all this is stored on a holder object, anything like `ClassName.#x()` will really have to be transformed to `base(ClassName, ClassName, holder)['x'].call(ClassName)`. This currently just outputs `base(ClassName, ClassName, holder)['x']()`.

Reply:
I think I just missed something about how the handlers are working and that made it way clearer. Thanks!